### DISCUSSION TITLE
[4] adding système submenu in backend

### DISCUSSION THREAD

**Discussion by micker** [joomla/joomla-cms#35289] [2. Cold Features]
Hello i like new admin menu with content component etc...
But i think WE Can enchance submenu of system to be more powerfull
I propose to add in système submenu
Template
Plugins
Manage extensions
Update
I think it will bé more interesting for integrator profil 
WE will win 1 click action
What do you think ?

**Comment by dgrammatiko** [comment]
> I think it will bé more interesting for integrator profil

Integrators could use the alternative menu, which is also more familiar for most people. This menu structure is oriented towards newcomers NON-admins

**Reply by micker** [reply]
Yes but adding isnt realy a problem because if WE use acl no problem to hide it that IS in core
Système menu IS the only one menu without submenu and that create more click for integrator user 
It will be more clear to have same ux in all menu and submenu

**Reply by dgrammatiko** [reply]
@micker there were some very clearly defined reasons why the system menu item didn't get submenus: **ALL** the tasks in the system view are not repeated tasks for any well-created site. Eg you update the site 5-8 times per year, same goes for the extensions. Enabling/disabling plugins/components/templates is not an everyday task and furthermore not one that anyone with permissions to the backend should be able to accomplish. All I'm saying is that if you're an integrator spend some time creating an XML for the menu that suits you copy it to your dev environment till the site is ready for publishing and then switch to the `easy mode` menu that Joomla comes as the default one...
